sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -459,6 +459,10 @@
               {
                 "label": "Partners Income",
                 "value": "partnerincomeappeal"
+              },
```

Comment:
In my story PR we talked about alphabetical order for the list of forms for  request a change.
As per the order this should be first. 

https://github.com/bcgov/SIMS/pull/2901#discussion_r1506660164